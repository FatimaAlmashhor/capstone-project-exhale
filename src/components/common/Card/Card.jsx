import React from 'react';
import PropTypes from 'prop-types';

export default function Card({ children }) {
  return (
    <div className="w-auto rounded overflow-hidden border hover:shadow-lg">
      {children}
    </div>
  );
}

Card.propTypes = {
  children: PropTypes.element.isRequired,
};
