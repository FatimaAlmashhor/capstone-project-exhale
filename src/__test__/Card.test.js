import React from 'react';
import renderer from 'react-test-renderer';
import Card from '../components/common/Card';

it('renders correctly', () => {
  const Test = <div>Hello</div>
  const tree = renderer
    .create(<Card>{Test}</Card> )
    .toJSON();
  expect(tree).toMatchSnapshot();
});