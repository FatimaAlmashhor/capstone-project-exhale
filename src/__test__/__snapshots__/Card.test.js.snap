// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="w-auto rounded overflow-hidden border hover:shadow-lg"
>
  <div>
    Hello
  </div>
</div>
`;
